### PR TITLE
Statistics Edits

### PR DESCRIPTION
**Pull request by bwerness**
Significant edit to the statistics section.  A few large modifications:
* Removed first figure and replaced with code that explained the same concept (hopefully clearer)
* Reordered the discussion of ways of evaluating estimators to improve flow
* Major rewrite on confidence intervals
* Expansion of discussion of statistical power
* Corrected incorrect implementation of bias/variance computation in code (now match exactly to machine precision)

I've attempted to incorporate the changes from Rachel's last PR, but given the large structural changes may have missed some (had to manually change it).  Rachel: please review to ensure I got every change you wanted to make!

### CODE REVIEW

File: chapter_appendix_math/statistics.md [hunk 1]
```diff
@@ -1,99 +1,122 @@
...
+# Compute true density
+xd = np.arange(np.min(xs),np.max(xs),0.01)
+yd = np.exp(-xd**2/2)/np.sqrt(2*np.pi)
+    
+# Plot the results
+d2l.set_figsize()
+d2l.plt.plot(xd,yd)
+d2l.plt.scatter(xs,ys)
+d2l.plt.axvline(x=0)
+d2l.plt.xlabel('x')
```
Comment (astonzhang) [inline chapter_appendix_math/statistics.md]: Can we feed xlabel, ylabel etc in d2l.plot():
See https://github.com/d2l-ai/d2l-en/blob/numpy2/d2l/d2l.py#L54

Comment (bwerness) [reply chapter_appendix_math/statistics.md]: Shortened.

File: chapter_appendix_math/statistics.md [hunk 2]
```diff
@@ -41,11 +41,9 @@ yd = np.exp(-xd**2/2)/np.sqrt(2*np.pi)
     
 # Plot the results
 d2l.set_figsize()
```
Comment (astonzhang) [inline chapter_appendix_math/statistics.md]: remove d2l.set_figsize()

d2l.plot() will invoke d2l.set_figsize()
https://github.com/d2l-ai/d2l-en/blob/numpy2/d2l/d2l.py#L58

File: chapter_appendix_math/statistics.md [hunk 3]
```diff
@@ -41,11 +41,9 @@ yd = np.exp(-xd**2/2)/np.sqrt(2*np.pi)
...
-d2l.plt.plot(xd,yd)
+d2l.plt.plot(xd,yd,'x','density')
```
Comment (astonzhang) [inline chapter_appendix_math/statistics.md]: Can we use d2l.plot, not d2l.plt.plot?

**Pull request merged**